lib/realtime/api.ex

```diff
@@ -6,7 +6,13 @@ defmodule Realtime.Api do
 
   import Ecto.Query
 
-  alias Realtime.{Repo, Api.Tenant, Api.Extensions, RateCounter, GenCounter, Tenants}
+  alias Realtime.Repo
+  alias Realtime.Repo.Replica
+  alias Realtime.Api.Tenant
+  alias Realtime.Api.Extensions
+  alias Realtime.RateCounter
+  alias Realtime.GenCounter
+  alias Realtime.Tenants
```

Comment:
I don't think this is necessary, realtime repo mainly uses a multi-alias style

Reply:
This was the thing I mentioned during the sync that makes it trickier to find module usage within the code (e.g. with Realtime.Api.Tenant it becomes way easier to find aliases vs Api.Tenant in one module and Tenant in another module)

Reply:
many apps use a multi alias style, such as [phoenix](https://github.com/phoenixframework/phoenix/blob/1c1d50e36a9a293c9649c1649cae57dfa386e328/lib/phoenix/socket.ex#L201)

Reply:
But, yeah, if it makes it harder to read, feel free to change that. However, I don't see any difference, just more lines